File: app/src/main/java/it/unical/mat/embasp/languages/asp/PredicateNotValidException.java
package it.unical.mat.embasp.languages.asp;

public class PredicateNotValidException extends Exception {

	/**
	 *
	 */
	private static final long serialVersionUID = 1L;

	public PredicateNotValidException() {
		super("Value of predicate is not valid");
	}

}

File: CSharp version/EmbASP/it/unical/mat/embasp/languages/asp/PredicateNotValidException.cs
using System;

namespace it.unical.mat.embasp.languages.asp
{
	public class PredicateNotValidException : Exception
	{

		private const long serialVersionUID = 1L;

		public PredicateNotValidException() : base("Value of predicate is not valid")
		{
		}

	}

}